src/Components/Header/Header.js

```diff
@@ -1,8 +1,12 @@
+import { useContext } from 'react';
+import { Context } from '../Wrapper';
```

Comment:
It's so cool that you got to gain experience with global state management using Context as well as setting up React Intl!